bug fix: Invalid validation marker in example in quick-start.md

Example in quick-start.md is as following:

    // GuestbookSpec defines the desired state of Guestbook
    type GuestbookSpec struct {
        // INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
        // Important: Run "make" to regenerate code after modifying this file

        // Quantity of instances
        // +kubebuilder:validation:Minimum=1
        // +kubebuilder:validation:Maximum=10
        Size int32 `json:"size"`

        // Name of the ConfigMap for GuestbookSpec's configuration
        // +kubebuilder:validation:MaxLength=15
        // +kubebuilder:validation:MinLength=1
        ConfigMapName string `json:"configMapName"`

        // +kubebuilder:validation:Enum=Phone,Address,Name
        Type string `json:"alias,omitempty"`
    }

Then run 'make install', I'll get error:

    /home/luyaozho/go/src/example/api/v1/guestbook_types.go:43:2: extra arguments provided: ",Address,Name" (at <input>:1:6)

The validation marker should be "// +kubebuilder:validation:Enum=Phone;Address;Name"

--- a/docs/book/src/quick-start.md
+++ b/docs/book/src/quick-start.md
@@ -107,7 +107,7 @@ type GuestbookSpec struct {
 	// +kubebuilder:validation:MinLength=1
 	ConfigMapName string `json:"configMapName"`
 
-	// +kubebuilder:validation:Enum=Phone,Address,Name
+	// +kubebuilder:validation:Enum=Phone;Address;Name
 	Type string `json:"alias,omitempty"`
 }
 

